.circleci/config.yml

```diff
@@ -1,25 +1,42 @@
 version: 2.0
 
+anchor_docker: &anchor_docker
+  docker:
+    - image: circleci/openjdk:8-jdk
+      environment:
+        TERM: dumb
+
+anchor_run_decrypt: &anchor_run_decrypt
+  name: Decrypt credentials.
+  command: |
+    openssl aes-256-cbc -md sha256 -d -in .circleci/it-service-account.json.enc \
+        -out .circleci/it-service-account.json -k "${SYSTEM_TESTS_ENCRYPTION_KEY}"
+
 jobs:
   datastore_it:
     working_directory: ~/googleapis
-    docker:
-      - image: circleci/openjdk:8-jdk
-        environment:
-          TERM: dumb
+    <<: *anchor_docker
     steps:
       - checkout
       - run:
-          name: Decrypt credentials.
-          command: |
-            openssl aes-256-cbc -md sha256 -d -in .circleci/it-service-account.json.enc \
-                -out .circleci/it-service-account.json -k "${SYSTEM_TESTS_ENCRYPTION_KEY}"
+          <<: *anchor_run_decrypt
       - run:
           name: Run integration tests for google-cloud-datastore
           command: ./utilities/verify_single_it.sh google-cloud-datastore
+  speech_it:
```

Comment:
In the future, will these IT jobs have to be explicitly listed?

Reply:
Yes, I'm not aware of a way to parameterize it in yaml.